.github/scripts/install_global_dependencies.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 
 # File path for the global dependencies
-GLOBAL_DEPENDENCIES_FILE="$PWD/global_dependencies"
+GLOBAL_DEPENDENCIES_FILE="$PWD/global_dependencies.json"
```

Comment:
Let's put all of these dependency things into Azle's root-level package.json file as we discussed.

Reply:
And let's add node, dfx, etc to try and put all dependency versions into one place

Reply:
https://github.com/demergent-labs/azle/issues/2095